Sources/SourceKitLSP/Swift/OpenInterface.swift

```diff
@@ -81,4 +110,46 @@ extension SwiftLanguageServer {
       }
     }
   }
+
+  private func _findUSR(
+    request: LanguageServerProtocol.Request<LanguageServerProtocol.OpenInterfaceRequest>, 
+    uri: DocumentURI, 
+    snapshot: DocumentSnapshot,
+    symbol: String?,
+    completion: @escaping (Swift.Result<FindUSRInfo, SKDError>) -> Void
+  ) {
+    guard let symbol = symbol else {
+      return completion(.success(FindUSRInfo(position: nil)))
+    }
+    let keys = self.keys
+    let skreq = SKDRequestDictionary(sourcekitd: sourcekitd)
+
+    skreq[keys.request] = requests.find_usr
+    skreq[keys.sourcefile] = uri.pseudoPath
+    skreq[keys.usr] = symbol
+
+    if let compileCommand = self.commandsByFile[uri] {
+      skreq[keys.compilerargs] = compileCommand.compilerArgs
+    }
+    
+    let handle = self.sourcekitd.send(skreq, self.queue) { result in
+      switch result {
+      case .success(let dict):
+          if let offset: Int = dict[keys.offset],
+            let position = snapshot.positionOf(utf8Offset: offset) {
+              return completion(.success(FindUSRInfo(position: position)))
+          } else {
+            return completion(.success(FindUSRInfo(position: nil)))
+          } 
```

Comment:
Should be indented by 2 spaces, I think.

Reply:
Do you use one of the swift formats to format the code?

Reply:
No, SourceKit-LSP is manual 2 space indentation